tests/resultspec_test.py

```diff
@@ -44,7 +44,7 @@ def archive(tmp_archive, mock_results):
 class TestResolveResultSpec:
     def test_existing_file(self, archive, mock_results):
         resultspec = archive._result_filename('a', '1', 'r1')
-        assert resolve_resultspec(archive, resultspec) == mock_results[0]
+        assert resolve_resultspecs(archive, resultspec)[0] == mock_results[0]
```

Comment:
Would `assert resolve_resultspecs(archive, resultspec) == mock_results` work?

Reply:
There should only be one result coming back from resolve_resultspecs in this case, but this isn't obvious from the test. I can clear this up.